### DISCUSSION TITLE
API文档也太难看了。是因为mac 的分辨率不适配吗

### DISCUSSION THREAD

**Discussion by kkrrrly** [milvus-io/milvus#6643] [Show and tell]
![image](https://user-images.githubusercontent.com/49581957/126259034-8ed40f40-281a-4e76-9ba6-ca84568a6b84.png)

**Comment by shanghaikid** [comment]
What is your screen resolution?

**Reply by kkrrrly** [reply]
Model:MacBook Pro (13-inch, 2017, Two Thunderbolt 3 ports) & Resolution: 2560 x 1600

**Reply by shanghaikid** [reply]
Thanks, we will adjust the styling soon.

**Reply by kkrrrly** [reply]
按类名排序的展现方式也不太友好，天知道我看了多久才看明白Param类和Param.Builder类的关系（v1.1.1）

**Reply by shanghaikid** [reply]
If this is better?  https://milvus-io.github.io/milvus-sdk-java/javadoc/index.html

**Reply by kkrrrly** [reply]
看起来好很多，风格也很API。
